cmd/support-logging/res/configuration.toml

```diff
@@ -24,13 +24,7 @@ File = ''
 
 [Databases]
   [Databases.Primary]
-  Host = 'localhost'
-  Name = 'logging'
-  Password = 'password'
-  Port = 27017
-  Username = 'logging'
-  Timeout = 5000
-  Type = 'mongodb'
+  Type = 'file'
```

Comment:
Why this change?

Reply:
Logger needs to default to a file when Redis is the default database.

Reply:
Ok, but there is no condition here.  This is changed for all use cases.